spec/models/maat_api/prosecution_case_spec.rb

```diff
@@ -43,11 +43,16 @@
   end
 
   it "has no crown_court_outcome" do
-    expect(prosecution_case.crown_court_outcome).to be_nil
+    expected = {
+      ccooOutcome: "CONVICTED",
+      caseEndDate: "2021-04-10",
+      appealType: nil,
+    }
+    expect(prosecution_case.crown_court_outcome).to eql(expected)
   end
 
   it "has an inactive" do
-    expect(prosecution_case.inactive).to eql("N")
+    expect(prosecution_case.inactive).to eql("Y")
```

Comment:
why the change here?

Reply:
Because I have set the default fixture file to being "CROWN" jurisdiction (for CCO) and so inactive calculates to "Y"